### ISSUE TITLE
Mac Zclassic wallet

### ISSUE THREAD

**Issue by koadowsett** [vaklinov/zclassic-swing-wallet-ui#3]
Hello i cannot tell which Zclassic wallet to download for Mac. Can anyone let me know please? Thanks!

**Comment by scubamysticool** [comment]
Find anything? I'm looking to get one too (mac user).

**Comment by koadowsett** [comment]
yep I just downloaded the latest version of the Eleos wallet, works fine but it did take about an hour and a half to initialise and fully sync. and you cant let your computer sleep while its trying to do the first initialise otherwise it will stop and you will have to start it again.

after i got the wallet i had to search for help exporting the private key.

To get the private key enter this command into console and use an existing address in the place of ADDRESSHERE:

./zcl-cli dumpprivkey ADRESSHERE

good luck champ
